rpcd/playbooks/elasticsearch.yml

```diff
@@ -54,6 +56,27 @@
       tags:
         - elasticsearch-pre-install
   roles:
-    - "elasticsearch"
+    - role: "ansible-elasticsearch"
+      es_instance_name: "openstack"
```

Comment:
Good catch, it was moved to group_vars for the artifact gating, I'll move them all there.